packages/material-ui/src/test-utils/describeConformance.js

```diff
@@ -146,12 +147,25 @@ function testRootClass(element, getOptions) {
   });
 }
 
+function testJsonRenderer(element) {
+  it('should render without errors in ReactTestRenderer', () => {
+    ReactTestRenderer.act(() => {
+      ReactTestRenderer.create(element, {
+        createNodeMock: node => {
+          return document.createElement(node.type);
+        },
+      });
+    });
+  });
+}
+
 const fullSuite = {
   componentProp: testComponentProp,
   mergeClassName: testClassName,
   propsSpread: testPropsSpread,
   refForwarding: describeRef,
   rootClass: testRootClass,
+  jsonRendered: testJsonRenderer,
```

Comment:
nit:
```suggestion
  reactTestRenderer: testReactTestRenderer,
```

json is just about serialization and not even used.

Reply:
Also need to add a comment block description to what it's for.

Reply:
Yeah this is one of those things that I've forgotten tomorrow about.